jekyll/_cci2/gh-bb-integration.md

```diff
@@ -107,7 +107,7 @@ The first two permissions require write-permission to a repo.
 
 This section provides an overview of the possible team and individual account choices available to meet various business needs:
 
-1. If an individual has a personal GitHub account, they will use it to log in to CircleCI and follow the project on CircleCI. Each 'collaborator' on that repository in GitHub is also able to follow the project and build on CircleCI when they push commits. If you invite a collaborator, but they do not appear in the Team page in CircleCI, you can refresh this  by clicking on the Resync with GitHub button from the Organization page.
+1. If an individual has a personal GitHub account, they will use it to log in to CircleCI and follow the project on CircleCI. Each 'collaborator' on that repository in GitHub is also able to follow the project and build on CircleCI when they push commits.
```

Comment:
Yes, and should we say that if you invite a collaborator, but they don't appear in the Team page, they should log out/log in to GitHub or CircleCI to see the updated list or something?